File: Bubblesort.py

def bubbleSort(arr):
    n=len(arr)
    
    for i in range(n):
        swapped=False
        for j in range(0,n-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
                swapped = True

        if (swapped == False):
            break






if __name__ == "__main__":
  
   arr=[90,78,60,50,30,20,1]
   bubbleSort(arr)
    
   print("Sorted Array: ")
   for i in range(len(arr)):
       print("%d" % arr[i] end=" ")

File: BubbleSort.java
import java.util.*;

class BubbleSort
{
  public static void main(String argu[])
  {
   int arr[]={90,22,34,10,29};
   int temp=0;
   int len=arr.length;
   for(int i=0;i<len;i++)
   {
     for(int j=0;j<len;j++)
     {
       if(arr[i]<arr[j])
       {
         temp=arr[i];
         arr[i]=arr[j];
         arr[j]=temp;
       }
     }
      System.out.println();
     for(int n:arr)
     {
       System.out.print(n+" ");
     }
   }

 
  }

}